lib/folio_record.rb

```diff
@@ -139,6 +139,10 @@ def as_json
     record
   end
 
+  def to_honeybadger_context
+    { hrid: }
```

Comment:
```suggestion
    { hrid:, instance_id: }
```

We already have (essentially) the hrid from traject's `record_inspect` output, but no harm duplicating it I suppose. Maybe the UUID would be nice to include?